tests/search/embedding/app_huggingface_embedder/schemas/doc.sd

```diff
@@ -0,0 +1,36 @@
+# Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+schema doc {
+
+  document doc {
+
+    field text type string {
+      indexing: index | summary
+    }
+
+  }
+
+  field embedding type tensor<float>(x[384]) {
+      indexing: input text | embed huggingface | attribute
+      attribute {
+          distance-metric: angular
+      }
+  }
+
+  rank-profile default {
+    inputs {
+      query(embedding) tensor(x[384])
+   }
+
+    first-phase {
+      expression: nativeRank
```

Comment:
You could for people stumbling uppon this change to closeness rank feature as nativRank does not take into account vector similarity/distance.

Reply:
Fixed in [5fe0bea](https://github.com/vespa-engine/system-test/pull/2853/commits/5fe0bea03ab28fa9812363d186a37ee88e3fdaa7)